packages/deploy-script-support/src/coreProposalBehavior.js

```diff
@@ -128,10 +149,10 @@ export const makeCoreProposalBehavior = ({
       }),
     );
 
-    // Evaluate the manifest for our behaviors.
+    // Evaluate the manifest.
     return runModuleBehaviors({
-      allPowers,
-      behaviors: manifestNS,
+      allPowers: powers,
```

Comment:
The reason I preferred `allPowers` over `powers` is that it really carries access to all the powers specified in the associated `permit.json`, and thus can be much more dangerous than just what `export const permits` describes.

Reply:
Good points. I've kept the name (since the actual scope of `powers` is not known by `behavior`), but added explanatory comments. PTAL.